messaging.rst

```diff
@@ -276,6 +276,148 @@ This should match `the Secret message in encoding/messages file <https://github.
+- The message should be sent by one of the channel participants.
+- The :term:`nonce` is increased by ``1`` with respect to the previous :term:`nonce`.
+- The message sender address should be the same as ``participant``.
+- The ``signature`` must be from the :term:`sender` of the request.
+
+Fields
+^^^^^^
+
++-------------------------------+---------------+----------------------------------------------------------------+
+| Field Name                    | Field Type    |  Description                                                   |
++===============================+===============+================================================================+
+|  cmdid                        | one byte      | Value 15 (indicating ``Withdraw Request``)                     |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  chain identifier             | uint256       | See :ref:`balance-proof-offchain                               |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  channel identifier           | uint256       | See :ref:`balance-proof-offchain`                              |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  token network address        | address       | See :ref:`balance-proof-offchain`                              |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  message identifier           | uint64        | An ID used in ``Delivered`` and ``Processed`` acknowledgements |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  participant                  | address       | The address of the withdraw requesting node                    |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  total_withdraw               | uint256       | The new monotonic ``total_withdraw`` value                     |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  expiration                   | uint256       | The block number at which withdraw request is no longer        |
+|                               |               | usable on-chain.                                               |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  nonce                        | uint64        | See :ref:`balance-proof-offchain`                              |
++-------------------------------+---------------+----------------------------------------------------------------+
+|  signature                    | bytes         | Elliptic Curve 256k1 signature                                 |
```

Comment:
I think instead of which curve is being used, what data is actually signed would be more informative here.

Reply:
Done